### ISSUE TITLE
Deploy backend on a cloud provider

### ISSUE THREAD

**Issue by HenriKCorrea** [right-purchase/management#5]
### Description

As a Developer, I want to deploy both API gateway and feedback service on heroku, So I can make the Right Purchase first release.

### Definition of Ready

The following stories must be done first:
- #1 
- #2 
- #3 
- #4 

### Acceptance Criteria

- Nice to have list:
-- CI/CD
-- TerraForm

### Definition of Done

- Backend services running on Heroku

**Comment by HenriKCorrea** [comment]
Changing Heroku cloud provider to Digital Ocean.
Reason: they don't provide kubernetes support
